### ISSUE TITLE
Wrong IP Address for app??

### ISSUE THREAD

**Issue by Tiger855** [slowscript/warpinator-android#143]
Where does Warpinator get it's IP address from? 
When unable to connect from Android phone to laptop, used "Manual connection" & noticed the displayed IP address was outside my network range & not the IP address of the phone.
Phone 10.0.0.204  app 10.5.0.2
I can ping the phone but obviously not the app.
Ran: nc -zvu 10.5.0.2 5353
        Connection to 10.5.0.2 5353 port [udp/mdns] succeeded!
Should the IP address be that of the phone?
Is there a way to correct the address?

**Comment by TwistedC0de** [comment]
You can install app called Turmux and run "ifconfig" command to see all the interfaces.

The "10.5.0.2" IP is likely coming from one of the interfaces.

"Is there a way to correct the address?"

I did not find any option for that in the Android app (desktop warpinator seems to have it) and faced a similar issue (VPN interface being used even when VPN was off).

Ended up cloning the repo and creating my own version of the app. Will create PR and see if that is accepted.

**Comment by TwistedC0de** [comment]
PR created #148

**Comment by slowscript** [comment]
Yes, the most likely cause is that a wrong network interface (which has the wrong IP address) is being selected. The referenced PR is included in version 1.8.3. There is also a new option to select preferred network interface, you probably want to select the one associated with WiFi (typically wlan0 or similar). Let me know if this solves your issue.
